content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md

```diff
@@ -136,7 +136,7 @@ This step should be done upon upgrading from one to another Kubernetes minor
 release in order to get access to the packages of the desired Kubernetes minor
 version.
 
-{{< tabs name="k8s_install_versions" >}}
+{{< tabs name="k8s_upgrade_versions " >}}
```

Comment:
Almost perfect! Just a small adjustment needed: there's an unnecessary space in the name.

```suggestion
{{< tabs name="k8s_upgrade_versions" >}}
```

Reply:
Thanks for review. I have committed the feedback suggestion.